lib/kotlin/cross-test-client/build.gradle.kts

```diff
@@ -33,10 +33,14 @@ val httpclientVersion: String by project
 val httpcoreVersion: String by project
 val logbackVersion: String by project
 val kotlinxCoroutinesJdk8Version: String by project
+val cliktVersion: String by project
 
 dependencies {
     implementation(platform("org.jetbrains.kotlin:kotlin-bom"))
     implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
+    // clikt is used to drive command line parsing and validation
+    // https://mvnrepository.com/artifact/com.github.ajalt.clikt/clikt
```

Comment:
I only just noticed these links. Please remove links to mvnrepository.com. It's a third-party indexing service, not affiliated with Maven Central, where these artifacts are being published. The maven coordinates are sufficient for people to find the artifact at search.maven.org, the official indexing service of Maven Central, or from their local maven repository acting as a proxy for Maven Central or hosting their local builds.